decidim-conferences/decidim-conferences.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "decidim-core", Decidim::Conferences.version
   s.add_dependency "decidim-meetings", Decidim::Conferences.version
-  s.add_dependency "wicked_pdf", "~> 2.1"
+  s.add_dependency "wicked_pdf", "~> 2.6"
```

Comment:
I understand this has been done because there was some failing spec? I can't seem to find it in the CI jobs history

Reply:
https://github.com/decidim/decidim/actions/runs/6467555236/job/17557824866
![image](https://github.com/decidim/decidim/assets/105683/bff4a262-e0f5-42d2-8593-9adbc9378ac3)

Reply:
Actually wicked_pdf version 2.7 adds the shakapacker support. Changed the dependency.

Reply:
Fixed in 38a4674287

Reply:
Ah, OK. I think I saw that run log, but I didn't see any reference to wicked in any traceback. That seems like it was fun to debug.